vscode-extension/src/devbox.ts

```diff
@@ -21,16 +22,17 @@ export async function devboxReopen() {
                 if (workspace.workspaceFolders) {
                     const workingDir = workspace.workspaceFolders[0].uri;
                     const dotdevbox = Uri.joinPath(workingDir, '/.devbox');
-                    try {
-                        // check if .devbox exists
-                        await workspace.fs.stat(dotdevbox);
-                    } catch (error) {
-                        //.devbox doesn't exist
-                        // running devbox shellenv to create it
-                        spawnSync('devbox', ['shellenv'], {
-                            cwd: workingDir.path
-                        });
-                    }
+                    progress.report({ message: 'Installing devbox packages...', increment: 25 });
+                    await setupDotDevbox(workingDir, dotdevbox);
+
+                    // setup required vscode settings
+                    progress.report({ message: 'Updating configurations...', increment: 50 });
+                    updateVSCodeConf();
+                    setTimeout(() => { }, 1000);
```

Comment:
Why do we need the `setTimeout`?

Reply:
ugh, that was from debugging, I'll remove it.